WooCommerce/Classes/Extensions/Site+PluginsURL.swift

```diff
@@ -0,0 +1,29 @@
+import Yosemite
+import Foundation
+
+private extension CardPresentPaymentsPlugin {
+    var setupURLSectionPath: String {
+        switch self {
+        case .wcPay:
+            return "woocommerce_payments"
+        case .stripe:
+            return "stripe"
+        }
+    }
+}
+
+/// Encapsulates the logic related to the provision of the Site URLs that point to plugins related content
+///
+extension Site {
+    /// Site's plugins section in wp-admin.
+    ///
+    var pluginsURL: String {
+        adminURL + "/plugins.php"
```

Comment:
The slash in the path shouldn't be needed

```suggestion
        adminURL + "plugins.php"
```

Reply:
Good catch, changed in https://github.com/woocommerce/woocommerce-ios/pull/6955/commits/2c1e3d4ad2b37e4f0e38e9245f9985b708b335fc